### ISSUE TITLE
Feedback about RMSNorm

### ISSUE THREAD

**Issue by ahmedelami** [pytorch/pytorch#197192]
There is the following issue on this page: https://docs.pytorch.org/docs/2.14/generated/torch.nn.modules.normalization.RMSNorm.html

<img width="767" height="390" alt="Image" src="https://github.com/user-attachments/assets/7690f746-9601-41ee-b734-ce8260ab7342" />

Should be a quick fix, I might do it if I have time.

cc @ezyang @svekars

**Comment by Rudra-G-23** [comment]
I can take a look at this and submit a fix.

**Comment by Rudra-G-23** [comment]
Fix #197213

**Comment by ahmedelami** [comment]
thanks so much! apologize for not taking care of it
